README.md

```diff
@@ -6,4 +6,7 @@ For installation guide and setup please visit each package's readme.
 
 [@bigcommerce/eslint-config](packages/eslint-config)
 
-[@bigcommerce/eslint-plugin](packages/eslint-plugin)
\ No newline at end of file
+[@bigcommerce/eslint-plugin](packages/eslint-plugin)
+
+## ESlint 9 Compatibility
+Due to a [breaking change](???) in ESLint 9, this configuration set currently **requires** ESLint 8. If you are installing this package directly (rather than transitively via [ng-build](https://github.com/bigcommerce/ng-build), you should use the same `eslint` version range as [package.json](./package.json).
```

Comment:
Since ng-build is a private repo, and our eslint config is public, should we change this verbiage a bit?

Reply:
My big concern is with external consumers of this library being confused here (if they click that link they'll hit a non-existent repo wall). We should treat this like an OSS repo (since it is) and only relay information that is pertinent to OSS consumers, whether that's external devs or us. 

Also, when you install ng-build you will hit a wall of peer dependency requests requiring you to explicitly install eslint, even if it is a transitive dependency of ng-build. Additionally, newer versions of PNPM (10+) will not hoist the transitive dependency so regardless of ng-build or not it should just be blank "transitive" statement instead.